core/subscriptions.go

```diff
@@ -26,8 +26,6 @@ func subscribe(channel string, client *client) {
 func unsubscribe(channel string, client *client) {
 	filtered := clients{}
 
-	// Probably can't handle unsubscribing from a non-existent channel, but
```

Comment:
alright, i removed it because if a person is not subscribed triggering unsubscribe should never end up there. if you wish i will put it back.